rebar.config

```diff
@@ -9,7 +9,7 @@
   {base16, ".*", {git, "git://github.com/goj/base16.git", "f78918e"}},
   {cuesport, ".*", {git, "git://github.com/esl/cuesport.git", "d82ff25"}},
   {redo, ".*", {git, "git://github.com/Wallapop/redo.git", "35a8d1c"}},
-  {exml, ".*", {git, "git://github.com/esl/exml.git", "2.4.1"}},
+  {exml, ".*", {git, "git://github.com/esl/exml.git", {ref, "7b5bfca"}}},
```

Comment:
Could you change exml ref to https://github.com/esl/exml/commit/d365533c193fe61fd211c74518b2136ef71b60ec ?